docs/gujarati.rst

```diff
@@ -0,0 +1,10 @@
+Gujarati
+******
+
+Gujarati is an Indo-Aryan language native to the Indian state of Gujarat. It
+is part of the greater Indo-European language family. Gujarati is descended from Old
+Gujarati (circa 1100–1500 AD). In India, it is the official language in the state of
+Gujarat, as well as an official language in the union territories of Daman and Diu and
+Dadra and Nagar Haveli.Gujarati is spoken by 4.5% of the Indian population, which amounts
+to 46 million speakers in India.Altogether, there are about 50 million speakers of Gujarati
+worldwide.(Source: `Wikipedia <https://en.wikipedia.org/wiki/Gujarati_language>`_) 
```

Comment:
:+1: , 
Please add Alphabets with some examples into docs see this http://docs.cltk.org/en/latest/greek.html#alphabet.